benchmark/framework/agency.py

```diff
@@ -0,0 +1,191 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This test module contains AEA/AEABuilder wrapper to make performance tests easy."""
+from threading import Thread
+from typing import Dict, List, Optional, Tuple, Type, Union
+
+from aea.aea import AEA
+from aea.aea_builder import AEABuilder
+from aea.configurations.base import ProtocolId, SkillConfig
+from aea.configurations.components import Component
+from aea.crypto.fetchai import FETCHAI
+from aea.mail.base import Envelope
+from aea.protocols.base import Message
+from aea.protocols.default.message import DefaultMessage
+from aea.protocols.default.serialization import DefaultSerializer
+from aea.skills.base import Handler, Skill, SkillContext
+
+
+class Agency:
+    """A testing wrapper to run and control an agent."""
+
+    def __init__(self, name: str = "my_eae", skills: List[dict] = None):
+        """Make an agency with optional name and skills."""
+        self.skills = skills or []
+        self.name = name
+
+        self.agent = self.make_agent(
+            self.name, [self.make_skill(**skill) for skill in self.skills]
+        )
+
+    def make_agent(
```

Comment:
please rename to `make_aea` for consistency with `AEA` class

Reply:
done